tests/bootstrap.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $loader = require dirname(__DIR__) . '/vendor/autoload.php';
-$loader->addPsr4('Bartlett\\', dirname(__DIR__) . '/src/Bartlett');
+$loader->addPsr4('Bartlett\\Monolog\\', dirname(__DIR__) . '/src');
```

Comment:
Why do you do this? Didn't you set this up in the composer.json?

Reply:
good catch. Will fix it

Reply:
commit eb1ef4d0bd65b67bf5e9e17299611f3e3b05bb7b fixed it